.github/workflows/build_and_test.yml

```diff
@@ -115,8 +100,6 @@ jobs:
             build \
             ${{ matrix.bazel-config }} \
             ${{ env.BAZEL_REMOTE_CACHE }} \
-            --verbose_failures \
-            -c dbg \
```

Comment:
Why did we remove these, here and below?

(I'd normally like to see such changes explained in the PR description)